libobs/media-io/video-io.c

```diff
@@ -136,6 +143,13 @@ static inline bool video_output_cur_frame(struct video_output *video)
 		struct video_input *input = video->inputs.array + i;
 		struct video_data frame = frame_info->frame;
 
+		uint32_t skip = input->fps_skipped_frames++;
+		if (input->fps_skipped_frames > input->fps_skip_frames)
+			input->fps_skipped_frames = 0;
```

Comment:
@tt2468: as far as `divisor` vs `skip_frames`, I currently find this terminology somewhat more straightforward, but if you have suggestions for better names that fit the divisor theme I'm not married to `skip_frames`

Reply:
I do worry the term "skipped frames" is one we already use both internally and in the Stats dialog, and would agree to prefer something else.

Reply:
Maybe 'interleave' or 'decimate' would be the alternative word to describe the feature.

Reply:
`interleave` also feels used up in the codebase, as far as `decimate` vs `divisor` I think I'd prefer `divisor` instead? `decimate` feels somewhat 'irregular'/'random' to me if that makes sense

Reply:
@WizardCM, @norihiro, @tt2468: I renamed things to the `divisor` naming scheme in [fixup! libobs: Allow configuring skip frames for encoders](https://github.com/obsproject/obs-studio/pull/9001/commits/5d5aaf9343f89e05b778b6d03901a9c79a51149b), thoughts?

Reply:
Rando hot take: maybe `frame_interval` an interesting synonym?

Reply:
Thinking about this again, I'm pretty confident that `frame_divisor` or `fps_divisor` is the way to go here. It's very commonly used for cases like this, and IMO it's just the most accurate term for the operation we're doing here.

Reply:
Closing since I just saw `frame_rate_divisor` and that seems adequate.